app/modules/suscribir/suscribible.rb

```diff
@@ -30,4 +31,23 @@ def desuscribe_a!(suscriptor, dominio_de_alta)
   def nombre_lista
     "#{self.class.name} id: #{id}#{ " (#{nombre})" if respond_to?(:nombre) }"
   end
+
+  def suscripciones_a_notificar(opciones = {})
+    todas = suscripciones.where.not(suscriptor_id: opciones[:excepto]).includes(:suscriptor).to_a
+    todas.delete_if do |suscripcion|
+      suscripcion.suscriptor.respond_to?(:emailable?) && !suscripcion.suscriptor.emailable?
+    end
+  end
+
+  def dame_datos_para_sendgrid(opciones = {})
+    datos = { emails: [], nombre: [], enlace_perfil_url: [], enlace_baja: [] }
+    suscripciones_a_notificar(opciones).each do |suscripcion|
+      datos[:emails] << suscripcion.email
+      datos[:nombre] << suscripcion.nombre_apellidos
+      usuario = suscripcion.suscriptor.try(:decorado)
+      datos[:enlace_perfil_url] << usuario.try(:enlace_perfil_url, 'Actualiza tu perfil')
```

Comment:
Maybe this is going to be different in the different main projects, so it could be a good idea to load it from `SuscribirPersonalizacion`. WDYT